threads/src/main/resources/org/jboss/as/threads/LocalDescriptions.properties

```diff
@@ -14,7 +14,7 @@ thread-factory=A thread factory (implementing java.util.concurrent.ThreadFactory
 thread-factory.add=Adds a thread factory
 thread-factory.name=The name of the created thread factory.
 thread-factory.group-name=Specifies the name of a  thread group to create for this thread factory.
-thread-factory.thread-name-pattern=The template used to create names for threads.  The following patterns may be used:\n\t%% - emit a percent sign\n\t%t - emit the per-factory thread sequence number\n\t%g - emit the global thread sequence number\n\t%f - emit the factory sequence number\n\t%i - emit the thread ID.
+thread-factory.thread-name-pattern=The template used to create names for threads.  The following patterns may be used:%n\t%% - emit a percent sign%n\t%t - emit the per-factory thread sequence number%n\t%g - emit the global thread sequence number%n\t%f - emit the factory sequence number%n\t%i - emit the thread ID.
```

Comment:
This one may not work. IIRC the strings from these resource bundles are not run through a `java.util.Formatter` like `String.format()`.